integration_tests/plugins/index.ts

```diff
@@ -16,6 +16,6 @@ export default (on: (string, Record) => void): void => {
 
     stubTokenVerificationPing: tokenVerification.stubPing,
 
-    stubIncomingMovements: welcome.stubIncomingMovements,
+    stubIncomingMovements: activeCaseLoadId => welcome.stubIncomingMovements(activeCaseLoadId),
```

Comment:
meh.. don't this needed changing. This was assigning a function to a function so shouldn't have needed to know about changes to how the function was being called

Reply:
reverted back